### DISCUSSION TITLE
Is there any way to get the code scanning feature of the Configuration Builder to ignore some folders?

### DISCUSSION THREAD

**Discussion by shrike71** [Avaiga/taipy#161] [Q&A] [answered]
Hi All,

Was putting the tutorial through it's paces, and very much liking what i was experiencing, until I hit one small issue. I have a habit of deploying a virtualenv (usually into a .venv/ folder) into the root folder of any application that I build, and then add it to my .gitignore file to exclude it from SCC.

I was doing the tutorial part here:

https://docs.taipy.io/en/release-2.1/getting_started/getting-started-core/step_01/ReadMe/

in vscode and when I clicked on the TASK visual object in the configuration builder and clicked the "function edit", I noticed that the builder was scanning for python files, presumably to introspect and provide hinting and autocompletion. However, the scanning process scans every folder sub to the application root, and it included the .venv/ folder. 21000+ files (and about 10 minutes) later it returned control to vscode! I only had taipy and it's dependencies installed, so this could have gone on for a lot longer. When I ditched the .venv/ inside the application root and created it parallel to it, the issue went away.

![image](https://github.com/Avaiga/taipy/assets/748514/1954136d-bb25-46f1-b798-4b1e8151ba97)

Is there any setting we can use to get the builder to ignore certain folders? Not a show stopper by any stretch of the imagination, but the .venv/ folder installed as part of the first run of a requirements.txt pattern is quite a common development pattern.

Keep up the good work. Certainly one of the most promising developer focussed low-code platforms that i've seen recently!

Shrike71

**Comment by FlorianJacta** [comment, marked as answer]
Thank you so much for your feedback! It has already been fixed. You will be able to ignore files in the next release of Taipy Studio.
